protected/humhub/modules/admin/controllers/ApprovalController.php

```diff
@@ -121,4 +163,73 @@ public function actionDecline($id)
             'approveFormModel' => $model
         ]);
     }
+
+    /**
+     * @return string|\yii\console\Response|\yii\web\Response
+     * @throws HttpException
+     * @throws \Throwable
+     * @throws \yii\base\InvalidConfigException
+     */
+    public function actionBulkActions()
+    {
+        /** @var string $action */
+        $action = Yii::$app->request->post('action');
+        /** @var array $usersId */
+        $usersId = Yii::$app->request->post('ids');
+
+        if (!$action) {
+            $this->view->error(Yii::t('AdminModule.controllers_ApprovalController', 'Please select an action (approve or decline)'));
```

Comment:
- Category: `AdminModule.base`
- Message: `No action has been selected.`

Reply:
This should normally not happen?

Reply:
You are right. I had written this code because previously it was a dropdown field that was used to select the action. Now that it's replaced by buttons, this code is useless, I deleted it.